data_processing/visualize_mass_spec.py

```diff
@@ -0,0 +1,80 @@
+import utils
+import os
+import argparse
+import pandas as pd
+import matplotlib.pyplot as plt
+import plotly.express as px
+
+
+
+
+def get_long_data(data_dir):
```

Comment:
Adding a docstring would be nice here! In general, all your functions would benefit from that type of inline documentation.

Reply:
I added this into the mass_spectrometry pull request. This probably shouldn't have been included in this request.